### PR TITLE
ECDC-4172 timepix pixel readout toa field offset incorrect

### PR DESCRIPTION
**Pull request by tbukovics**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

### Nominate for Group Code Review

- [ ] Nominate for code review

### CODE REVIEW

File: src/modules/timepix3/dto/TimepixDataTypes.h [hunk 1]
```diff
@@ -248,14 +248,14 @@ struct PixelReadout {
    * @param dCol The digital column of the pixel.
    * @param sPix The sub pixel index of the pixel.
    * @param pix The pixel index within the sub pixel.
+   * @param toa The Time-of-Arrival value of the pixel.
    * @param ToT The Time-over-Threshold value of the pixel.
    * @param fToA The fine Time-of-Arrival value of the pixel.
-   * @param toa The Time-of-Arrival value of the pixel.
    * @param spidrTime The SPIDR timestamp of the pixel.
    */
```
Comment (mortenjc) [inline src/modules/timepix3/dto/TimepixDataTypes.h:255]: Please remember to use the C++ comment style for Doxygen ;-)

/// \brief  etc.

File: src/modules/timepix3/readout/DataParser.cpp [hunk 2]
```diff
@@ -7,6 +7,8 @@
 //===----------------------------------------------------------------------===//
 
 #include <common/debug/Trace.h>
+#include <iostream>
+#include <ostream>
```
Comment (mortenjc) [inline src/modules/timepix3/readout/DataParser.cpp:11]: Are you sure you need these?

File: src/modules/timepix3/readout/DataParser.h [hunk 3]
```diff
@@ -33,7 +33,7 @@ namespace Timepix3 {
 #define PIXEL_DCOL_OFFSET   52
 #define PIXEL_SPIX_OFFSET   45
 #define PIXEL_PIX_OFFSET    44
-#define PIXEL_TOA_OFFSET    28
+#define PIXEL_TOA_OFFSET    30
```
Comment (mortenjc) [inline src/modules/timepix3/readout/DataParser.h:36]: I'm wondering  if this should have been caught in a unit test?

**Review by mortenjc — APPROVED** [review]

File: src/modules/timepix3/readout/DataParser.h (hunk 3 above)
Comment (tbukovics) [reply src/modules/timepix3/readout/DataParser.h:36]: It should, but the unit test was also incorrect made after the code assume the code is correct. It was a mistake during the project handover.

**Pull request merged**